src/humanize/i18n.py

```diff
@@ -123,7 +123,7 @@ def _ngettext(message: str, plural: str, num: int) -> str:
     Returns:
         str: Translated text.
     """
-    return get_translation().ngettext(message, plural, num)
+    return get_translation().ngettext(message, plural, int(num))
```

Comment:
Let's skip this, there may be some cases where we don't want to cast to float (re: https://code.djangoproject.com/ticket/27486). Let's just handle `precisedelta` in this PR, and if the warning comes up for other functions, we can deal with those as appropriate.

```suggestion
    return get_translation().ngettext(message, plural, num)
```

Reply:
Sure